src/_cffi_src/openssl/bio.py

```diff
@@ -11,37 +11,8 @@
 TYPES = """
 typedef struct bio_st BIO;
```

Comment:
wtf? ok, that's fine I guess, maybe send a PR to upstream that seems like
an oversight?

On Fri, Apr 29, 2016 at 9:31 AM, Paul Kehrer <notifications@github.com>
wrote:

> In src/_cffi_src/openssl/bio.py
> <https://github.com/pyca/cryptography/pull/2881#discussion_r61577736>:
>
> > @@ -11,37 +11,8 @@
> >  TYPES = """
> >  typedef struct bio_st BIO;
>
> CRYPTO_mem_leaks(struct bio_st *); is (obnoxiously) a function signature.
> We can do this if we want to move this function to MACROS and change its
> definition though.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2881/files/5c6fdf6dc6bdfa65259a69cd2f10f244e53c6fe8#r61577736>
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6